<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ger_cli</class>
 <widget class="QDialog" name="ger_cli">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_editar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw_clientes">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>781</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Filtrar por:</string>
   </property>
   <widget class="QRadioButton" name="rb_nome">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome Cliente</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Cliente</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="txt_filtrar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_filtrar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
